src/main/java/ch/interlis/iox_j/validator/Validator.java

```diff
@@ -333,6 +337,40 @@ public void validate(ch.interlis.iox.IoxEvent event) {
 			}
 		}
 	}
+    private void validateInconsistentIliAndXMLVersion(ch.interlis.iox.IoxEvent event) {
+        String versionControl = config.getValue(CONFIG_DO_XTF_VERSIONCONTROL);
+        if (versionControl != null && event instanceof XtfStartTransferEvent) {
+            Model model = getModelFromTransferDesc();
+            String iliVersion = model.getModelVersion();
+            String modelName = model.getName();
+            XtfStartTransferEvent startTransferEvent = (XtfStartTransferEvent) event;
+            Collection<IomObject> headerObjValues = startTransferEvent.getHeaderObjects().values();
+            List<IomObject> headerObjects = new ArrayList<IomObject>(headerObjValues);
+            for(IomObject currentObj : headerObjects) {
+                String currentVersion = currentObj.getattrvalue("version");
+                if (!(currentVersion.equals(iliVersion))) {
+                    String versionInfoMessage = "The Iliversion (" + iliVersion + ") in a Model (" + modelName
```

Comment:
The VERSION in model (" + modelName  + ") and transferfile do not match (" + iliVersion + "!=" + currentVersion + ")";